src/main/resources/application.properties.EXAMPLE

```diff
@@ -4,15 +4,24 @@ app.name=cbioportal
 # Spring Boot Properties 2.7.14
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
 
+#Clickhouse Enabled
+# Set to True to enable Clickhouse (Features)
+#clickhouse_mode=false
+
 # database
-spring.datasource.mysql.url=jdbc:mysql://localhost:3306/cbioportal?useSSL=false
-spring.datasource.mysql.username=cbio
-spring.datasource.mysql.password=P@ssword1
-spring.datasource.mysql.driver-class-name=com.mysql.jdbc.Driver
-spring.datasource.clickhouse.url=jdbc:ch://localhost:8443/cbioportal
-spring.datasource.clickhouse.username=dummy
-spring.datasource.clickhouse.password=dummy
-spring.datasource.clickhouse.driver-class-name=com.clickhouse.jdbc.ClickHouseDriver
+
```

Comment:
@haynescd @inodb maybe we should hide this configuration stuff for the time being?